src/pages/stablecoins.js

```diff
@@ -0,0 +1,882 @@
+              </DaiButtonRow>
+            </div>
+          </StyledLeftColumn>
+          <Hero
+            fluid={data.dailarge.childImageSharp.fluid}
+            alt="The Dai logo"
+          />
+        </DaiBanner>
+      </Content>
+      <Content>
+        <H2 id="explore">How to get stablecoins</H2>
+      </Content>
+      <FullWidthContainer>
+        <StablecoinAccordion />
+      </FullWidthContainer>
+      <Divider />
+      <Content>
+        <StyledCalloutBanner
+          title="Use your stablecoins"
+          description="Check out Ethereum’s dapps – stablecoins are often more useful for everyday transactions."
+          image={data.doge.childImageSharp.fluid}
+          maxImageWidth={600}
+          alt="Illustration of a doge."
+        >
+          <div>
+            <ButtonLink to="/dapps/">Explore dapps</ButtonLink>
+          </div>
+        </StyledCalloutBanner>
+        <TwoColumnContent>
+          <LeftColumn>
+            <H2>Save with stablecoins</H2>
+            <p>
+              Stablecoins often have an above-average interest rate because
+              there’s a lot of demand for borrowing them. There are dapps out
+              there that let you earn interest on your stablecoins in real time.
+              In the background, you're supplying tokens for potential borrowers
+              but you can withdraw your tokens and your interest at any time.
+            </p>
+          </LeftColumn>
+          <RightColumn>
+            <Warning>
+              <H2>Always do your own research</H2>
+              Ethereum is a new technology and most applications are new. Make
+              sure you're aware of the risk and only deposit what you can afford
+              to lose.
+            </Warning>
+          </RightColumn>
+        </TwoColumnContent>
+        <StyledCardGrid>
+          {dapps.map((dapp, idx) => {
+            return (
+              <ProductCard
+                key={idx}
+                background={dapp.background}
+                url={dapp.url}
+                alt={dapp.alt}
+                image={dapp.image}
+                name={dapp.name}
+              >
+                <p>{dapp.description}</p>
+              </ProductCard>
+            )
+          })}
+        </StyledCardGrid>
+      </Content>
+      <Divider />
+      <Content id="how">
+        <H2>How they work: types of stablecoin</H2>
+        <StablecoinBoxGrid items={features} />
+      </Content>
+      <H2>Top stablecoins by market capitalisation</H2>
+      <p>
+        Market capitalisation is <code>total supply x value</code>.
+      </p>
+      <TableContent>
+        <SimpleTable
+          column1="Currency"
+          column2="Market cap"
+          column3="Type"
+          content={table}
+        />
+      </TableContent>
+    </Page>
+  )
+}
+
+export default StablecoinsPage
+
+export const query = graphql`
+  query {
+    stablecoins: file(relativePath: { eq: "stablecoins/hero.png" }) {
```

Comment:
Nice hero image! Do we have permission to use this?

Reply:
;)